actors/market/src/state.rs

```diff
@@ -528,15 +529,17 @@ where
     // Return true when the funds in escrow for the input address can cover an additional lockup of amountToLock
     pub(super) fn balance_covered(
         &self,
-        addr: Address,
```

Comment:
nit: Since this is keyed by address it would make sense for this code to expect an address, not change its internal logic to better adapt to the caller.